dumpling/export/config.go

```diff
@@ -621,9 +621,10 @@ const (
 	// DefaultTableFilter is the default exclude table filter. It will exclude all system databases
 	DefaultTableFilter = "!/^(mysql|sys|INFORMATION_SCHEMA|PERFORMANCE_SCHEMA|METRICS_SCHEMA|INSPECTION_SCHEMA)$/.*"
 
-	defaultDumpThreads        = 128
-	defaultDumpGCSafePointTTL = 5 * 60
-	defaultEtcdDialTimeOut    = 3 * time.Second
+	// set a large number to avoid to block task dispatch
+	defaultTaskChannelCapacity = 10000
```

Comment:
What if chunk task length is greater than 10000?
BTW, a large size channel doesn't obay golang's development rules, right? @lance6716

Reply:
I agree

https://github.com/uber-go/guide/blob/master/style.md#channel-size-is-one-or-none

Reply:
> What if chunk task length is greater than 10000?

This indeed a problem that we can't make sure the chunk quantity is in a defined scope. But I didn't think of a better way to not block chunks count, meanwhile not postpone the data dump.